core/src/test/resources/crypto-on-accumulo-site.xml

```diff
@@ -76,8 +76,12 @@
       <value>org.apache.accumulo.core.security.crypto.DefaultCryptoModule</value>
     </property>
     <property>
+      <name>crypto.security.provider</name>
+      <value>SunJCE</value>
```

Comment:
Does this test actually need SunJCE, specifically?

Reply:
I wanted to include a provider to exercise code in unit tests for a configured provider.